tools/tiled_ensemble/post_processing/__init__.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
I am confused about this. Is this still relevant?

Reply:
No it's not, I'll remove this.